client/src/components/ReadingList/ManualReadingListAdder.tsx

```diff
@@ -0,0 +1,40 @@
+import { CheckCircleOutlined, CloseCircleOutlined, PlusOutlined } from '@ant-design/icons';
+import { Button, Input } from 'antd';
+import React, { useState } from 'react';
+import { useAppSelector } from '../../hooks/reduxHooks';
+import { blankPaper } from '../../templates';
+import { useUpdateReadingListFunc } from './hooks';
+
+export default function ManualReadingListAdder(): JSX.Element {
+  const [isEditing, setIsEditing] = useState(false);
+  const [title, setTitle] = useState('');
+  const readingList = useAppSelector((state) => state.readingList);
+
+  const { updateReadingListFunc } = useUpdateReadingListFunc();
+
+  const submitItem = () => {
+    setTitle('');
+    const newReadingList = readingList.concat({ ...blankPaper, title });
+    updateReadingListFunc(newReadingList);
+  };
+
+  return (
+    <div className="reading-list-adder">
+      {!isEditing ? (
+        <Button onClick={() => setIsEditing(true)} className="reading-list-adder__startButton">
+          Add to Reading List <PlusOutlined />
+        </Button>
+      ) : (
+        <div className="reading-list-adder__form">
+          <Input placeholder="Paper title" value={title} onChange={(e) => setTitle(e.target.value)} />
+          <Button aria-label="submit" onClick={submitItem} disabled={title === ''}>
+            <CheckCircleOutlined />
+          </Button>
+          <Button aria-label="cancel" danger onClick={() => setIsEditing(false)}>
+            <CloseCircleOutlined />
+          </Button>
+        </div>
```

Comment:
Briefly considered breaking this out into a separate element, but I think it's concise enough that it's okay here as-is

Reply:
That's true, though one advantage of making it separate is that its state naturally follows its lifecycle. You could probably avoid `setTitle('')` if you just unmounted an isolated component when it isn't in use.

Compromise: make it a separate component that's declared here and not exported?